book/src/SUMMARY.md

```diff
@@ -19,7 +19,7 @@
 - [Teams](./teams/README.md)
   - [Core Concepts](./teams/core-concepts.md)
   - [Teams Manifest](./teams/manifest.md)
-  - [Teams Toolkit](./teams/m365-toolkit.md)
+  - [M365 Agents Toolkit](./teams/m365-toolkit.md)
```

Comment:
Should we label is "M365 Agents Tookit (ATK)"? Might be easier if we introduce it as ATK once and then just use that.

Reply:
We were asked not to use the abbreviation ATK in official docs capacity.

Reply:
Oh okay - but aren't we going to use atk in the cli? Won't that be confusing if we don't document what atk is anywhere?